src/client.js

```diff
@@ -71,6 +71,8 @@ class Client extends events {
 
 		let disconnectReason;
 		let joinErr;
+		let wsReason;		
```

Comment:
Why

Reply:
to log the disconnect reason provided by skribbl.io